app/workers/world_location_news_worker.rb

```diff
@@ -2,7 +2,7 @@ class WorldLocationNewsWorker < WorkerBase
   attr_accessor :world_location
 
   def perform(world_location_id)
-    self.world_location = WorldLocation.find(world_location_id)
+    self.world_location = WorldLocation.find(world_location_id).reload
```

Comment:
Do we still need the `reload` here?  I think we only added it to test something.

Reply:
Oh yeah good shout, will remove